modules/dshow/dshow.cpp

```diff
@@ -43,10 +45,11 @@ IID IID_ISampleGrabberCB = {
 
 #include "qedit.h"
 
+/*
 const CLSID CLSID_SampleGrabber = { 0xc1f400a0, 0x3f08, 0x11d3,
   { 0x9f, 0x0b, 0x00, 0x60, 0x08, 0x03, 0x9e, 0x37 }
 };
-
+*/
```

Comment:
Really not sure on this one. 
I am using `qedit.h` from here: https://github.com/thegrandpoobah/dsgrab/blob/master/qedit.h
and it has `CLSID_SampleGrabber` defined. The thing is that even though this is an extract from DirectX SDK, in this repo it has GPL preamble in the file. So I'd prefer not to use it at all.

As far as I understand `qedit.h` was a part of Windows SDK/ DirectX SDK, but now it's removed (see https://social.msdn.microsoft.com/Forums/windowsdesktop/en-US/2ab5c212-5824-419d-b5d9-7f5db82f57cd/qedith-missing-in-current-windows-sdk-v70?forum=windowsdirectshowdevelopment or https://blogs.msdn.microsoft.com/karinm/2010/01/15/where-is-qedit-h/)

Do you think we could incorporate required CLSIDs and other stuff directly into `dshow.cpp` or new header file in the project?